src/reconstruction/CoreModelCoupled.jl

```diff
@@ -362,40 +362,96 @@ function change_coupling_bounds!(
     end
 end
 
-"""
-    change_bounds!(
-        model::CoreModelCoupled,
-        rxns::Vector{Int};
-        xl::V = Float64[],
-        xu::V = Float64[],
-    )
+@doc @_change_bound_s_bang("CoreModelCoupled", "rxn_idxs", "Vector{Int}", "[2, 43]", true, true)
+function change_bounds!(
+    model::CoreModelCoupled,
+    reaction_idxs::Vector{Int};
+    lower_bounds = fill(-_constants.default_reaction_bound, length(rxns)),
+    upper_bounds = fill(_constants.default_reaction_bound, length(rxns)),
+)   
+    for (rxn_idx, lb, ub) in zip(reaction_idxs, lower_bounds, upper_bounds)
+        change_bound!(model, rxn_idx; lower_bound=lb, upper_bound=ub)
```

Comment:
Let's try with
```
change_bound!.(Ref(model), rxn_idxs, ........)
```

Reply:
This doesn't seem to work with kwargs...
```julia
julia> g(x,y) = x + y
g (generic function with 1 method)

julia> g.(Ref(1), [1,2,3])
3-element Vector{Int64}:
 2
 3
 4
```
but
```julia
julia> g(x,y;z=[0,0,0]) = x + y + z
g (generic function with 1 method)

julia> g.(Ref(1), [1,2,3]; z=[1,2,3])
ERROR: MethodError: no method matching +(::Int64, ::Vector{Int64})
```